llmproxy/provider/google/vertexai.py

```diff
@@ -24,10 +46,41 @@
             "Financial Task": 2,
             "Content Recommendation Task": 1,
         },
+        "chat-bison": {
+            "Code Generation Task": 1,
+            "Text Generation Task": 1,
+            "Translation and Multilingual Applications Task": 1,
+            "Natural Language Processing Task": 1,
+            "Conversational AI Task": 2,
+            "Educational Applications Task": 1,
+            "Healthcare and Medical Task": 2,
+            "Legal Task": 2,
+            "Financial Task": 2,
+            "Content Recommendation Task": 1,
+        },
+        "gemini-pro": {
+            "Code Generation Task": 3,
+            "Text Generation Task": 3,
+            "Translation and Multilingual Applications Task": 3,
+            "Natural Language Processing Task": 3,
+            "Conversational AI Task": 3,
+            "Educational Applications Task": 3,
+            "Healthcare and Medical Task": 3,
+            "Legal Task": 3,
+            "Financial Task": 3,
+            "Content Recommendation Task": 3,
+        },
     }
 }
 
 
+class VertexAIModel(str, BaseEnum):
```

Comment:
We don't use model enums anymore since the check occurs in the setup, so you have to remove this as well.

Reply:
removed